.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  run-tests-for-nodejs:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 16.x]
```

Comment:
I think testing the immediate previous stable version, current stable version and dev version should be sufficient. For Node.js at May 2022, that are 14.x, 16.x and 18.x respectively.